src/main/java/com/amazonaws/services/dynamodbv2/streamsadapter/StreamsMultiLangDaemon.java

```diff
@@ -58,7 +59,7 @@ public static void main(String[] args) {
         ExecutorService executorService = config.getExecutorService();
 
         Worker worker = new StreamsWorker(
-                config.getRecordProcessorFactory(),
+                (IRecordProcessorFactory) config.getRecordProcessorFactory(),
```

Comment:
Can you please explain what is the motivation behind this explicit casting? It already used to be of type 'IRecordProcessorFactory' correct?